### ISSUE TITLE
react-router 4.4.0 - Invariant Violation: You should not use <Route> or withRouter() outside a <Router>

### ISSUE THREAD

**Issue by mmuller99** [chacestew/react-router-tabs#19]
Hi @chacestew , first of all, thanks for writing this useful library. The `RoutedTabs` components seem to fail after upgrading to react-router 4.4.0

**Comment by chacestew** [comment]
Hi @mmuller99 - thanks for raising this. Can you confirm if this occurs even if your app is wrapped in `<BrowserRouter />` higher up?

**Comment by megothss** [comment]
I can confirm that the problem occurs after upgrading to react-router 5.0.0 even with a `<BrowserRouter />` higher up in the component chain.

The following stack trace is produced in my aplication:

```
The above error occurred
 in the <Route> component:
 in Route (created by NavTab)
 in NavTab (at index.jsx:37)
 in div (created by RoutedTabs)
 in RoutedTabs (at index.jsx:35)
 in Scopes (at context.jsx:8)
 in ScopeAwareComponent (at pov.jsx:13)
 in PovAwareComponent
 in Route (created by withRouter(PovAwareComponent))
 in withRouter(PovAwareComponent) (at index.jsx:84)
 in div (at index.jsx:83)
 in div (at index.jsx:82)
 in CreditoPF (at index.jsx:69)
 in e (at index.jsx:12)
 in Route (at index.jsx:9)
 in Unknown (at index.jsx:56)
 in Switch (at index.jsx:54)
 in div (created by e)
 in e (at index.jsx:53)
 in Panels (at pov.jsx:13)
 in PovAwareComponent (at app.jsx:147)
 in Route (at app.jsx:147)
 in Switch (at app.jsx:136)
 in div (at app.jsx:116)
 in App (created by Connect(App))
 in Connect(App) (at fetch.jsx:115)
 in Get(App)
 in Route (created by withRouter(Get(App)))
 in withRouter(Get(App)) (at index.jsx:54)
 in Frames (created by Connect(Frames))
 in Connect(Frames) (at index.jsx:47)
 in Route (at index.jsx:44)
 in Provider (at index.jsx:43)
 in Router (created by BrowserRouter)
 in BrowserRouter (at index.jsx:42) 

 Consider adding an error boundary to your tree to customize error handling behavior. Visit https://fb.me/react-error-boundaries to learn more about error boundaries.
```

**Comment by chacestew** [comment]
It looks like the same recent issue as https://github.com/ReactTraining/react-router/issues/6634, which is related to changes in version 4.4/5 of react-router.

A quick fix is to pin your project's versions of react-router-dom and/or react-router to `4.3.0`.

I am going to try find some time to look at this soon. If anyone else has insights, please share.

**Comment by gaurav-quasar** [comment]
I am facing the same issue.

**Comment by chacestew** [comment]
I have created a new release under the `@next` tag that should fix this issue. @mmuller99, @megothss, and @gaurav-quasar, please would you test in your projects using

 `npm install react-router-tabs@next` 

and let me know the result.

**Comment by mmuller99** [comment]
@chacestew , tested and working 👍 Thanks a lot

**Comment by chacestew** [comment]
This fix is now released with the latest version: `1.3.2`
